index.js

```diff
@@ -97,7 +98,7 @@ router
         "sampleRate": 1,
         "tags": ["cloudflareErrorType:" + ctx.query['cloudflareErrorType']]
       })
-      console.log('Cloudflare Error -- Type: ' + ctx.query['cloudflareErrorType'] + ' -- Ray ID: ' + ctx.query['rayID'] + ' -- Client IP: ' + ctx.query['clientIP'])
+      console.log(`Cloudflare Error -- Type: ${ctx.query['cloudflareErrorType']} -- Ray ID: ${ctx.query['rayID']} -- Client IP: ${ctx.query['clientIP']}`)
```

Comment:
When we get an error, we will create a `console.log` in volley. Is there any other type of alerting that happens, or will tracking these errors mean reading the call stacks?